<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CrewControls</class>
 <widget class="QWidget" name="CrewControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>295</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="verticalSpacing">
    <number>2</number>
   </property>
   <item row="2" column="2">
    <widget class="QPushButton" name="add_wound_btn">
     <property name="text">
      <string>Add Wound</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLineEdit" name="name">
     <property name="placeholderText">
      <string>Crew Member Name</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QSlider" name="wounds">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QSlider::groove:horizontal {
border: 1px solid #bbb;
background: white;
height: 4px;
border-radius: 4px;
}

QSlider::sub-page:horizontal {
background-color: blue;
border: 1px solid #777;
height: 4px;
border-radius: 4px;
}

QSlider::add-page:horizontal {
background: #fff;
border: 1px solid #777;
height: 4px;
border-radius: 4px;
}

QSlider::handle:horizontal {
background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
    stop:0 #eee, stop:1 #ccc);
border: 1px solid #777;
width: 8px;
margin-top: -6px;
margin-bottom: -6px;
border-radius: 4px;
}

QSlider::sub-page:horizontal:disabled {
background: #bbb;
border-color: #999;
}

QSlider::add-page:horizontal:disabled {
background: #eee;
border-color: #999;
}

QSlider::handle:horizontal:disabled {
background: #eee;
border: 1px solid #aaa;
border-radius: 4px;
}</string>
     </property>
     <property name="maximum">
      <number>3</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBothSides</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="score_red_btn">
     <property name="text">
      <string>Score Red Hit</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QSpinBox" name="reds_spin">
     <property name="suffix">
      <string> reds</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" rowspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QGroupBox" name="action_grp">
     <property name="title">
      <string>Turn Actions</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QRadioButton" name="shoot_radio">
        <property name="text">
         <string>Shoot</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="gun_selection_shoot"/>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="reload_radio">
        <property name="text">
         <string>Reload</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="gun_selection_reload"/>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="unjam_radio">
        <property name="text">
         <string>Unjam</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="gun_selection_unjam"/>
      </item>
      <item row="4" column="0">
       <widget class="QRadioButton" name="observe_radio">
        <property name="text">
         <string>Observe</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QRadioButton" name="drop_bomb_radio">
        <property name="text">
         <string>Drop Payload</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="remaining_bombs_lbl">
        <property name="text">
         <string>&lt;remaining_bombs&gt;</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QRadioButton" name="custom_radio">
        <property name="text">
         <string>Custom</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="custom_action_line_edit"/>
      </item>
      <item row="7" column="0" colspan="2">
       <widget class="QPushButton" name="log_action">
        <property name="toolTip">
         <string>Note only one action can be logged per crew per turn</string>
        </property>
        <property name="text">
         <string>Log Action</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QFrame" name="shoot_frame">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>9</number>
         </property>
         <property name="rightMargin">
          <number>9</number>
         </property>
         <property name="bottomMargin">
          <number>9</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>Current Box:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="QLabel" name="ammo_box_current">
           <property name="text">
            <string>20/20 shots remain</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>Spare Boxes:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="3">
          <widget class="QFrame" name="frame">
           <property name="autoFillBackground">
            <bool>true</bool>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <property name="leftMargin">
             <number>5</number>
            </property>
            <property name="topMargin">
             <number>5</number>
            </property>
            <property name="rightMargin">
             <number>5</number>
            </property>
            <property name="bottomMargin">
             <number>5</number>
            </property>
            <property name="spacing">
             <number>3</number>
            </property>
            <item row="2" column="3">
             <widget class="Line" name="line_20">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_4">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Range</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0" colspan="9">
             <widget class="Line" name="line_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_13">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Fire Base</string>
              </property>
             </widget>
            </item>
            <item row="2" column="5">
             <widget class="Line" name="line_21">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
            <item row="0" column="7">
             <widget class="Line" name="line_18">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
            <item row="0" column="8">
             <widget class="QLabel" name="label_12">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>0</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="2" column="8">
             <widget class="QLabel" name="fire_base_0">
              <property name="text">
               <string>0</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="label_8">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>3</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="2" column="4">
             <widget class="QLabel" name="fire_base_2">
              <property name="text">
               <string>0</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QLabel" name="fire_base_3">
              <property name="text">
               <string>0</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="4">
             <widget class="QLabel" name="label_9">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>2</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="Line" name="line_19">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
            <item row="2" column="7">
             <widget class="Line" name="line_22">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
            <item row="2" column="6">
             <widget class="QLabel" name="fire_base_1">
              <property name="text">
               <string>0</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="Line" name="line_5">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="Line" name="line_8">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
            <item row="0" column="6">
             <widget class="QLabel" name="label_10">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>1</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="5">
             <widget class="Line" name="line_17">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="3" column="0" colspan="3">
          <widget class="QSlider" name="burst_len">
           <property name="maximum">
            <number>2</number>
           </property>
           <property name="pageStep">
            <number>1</number>
           </property>
           <property name="value">
            <number>2</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QPushButton" name="short_burst_btn">
           <property name="text">
            <string>Short</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QPushButton" name="med_burst_btn">
           <property name="text">
            <string>Medium</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QPushButton" name="long_burst_btn">
           <property name="text">
            <string>Long</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="QLabel" name="ammo_total">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>shoot_radio</sender>
   <signal>toggled(bool)</signal>
   <receiver>shoot_frame</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>146</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
